.gitignore

```diff
@@ -19,4 +19,9 @@ Homestead.json
 Homestead.yaml
 
 npm-debug.log
-yarn-error.log
\ No newline at end of file
+yarn-error.log
+
+/app/Models/City.php
+/app/Models/Country.php
+/app/Models/State.php
+/app/Models/CountryStateCityTableSeeder.php
```

Comment:
Remove this from PR cuz when we are change something in this model then it will be problem